### DISCUSSION TITLE
I have a question regarding the Keyball61

### DISCUSSION THREAD

**Discussion by Forseti4** [Yowkees/keyball#400] [Q&A] [answered]
Hello, I have recently purchased and assembled a Keyball61. 
The left board works properly, but when I connect the right board, 'dfg' is continuously input and the LED does not turn on normally. 
I was wondering if you might know a solution to this issue
![20231029_202210](https://github.com/Yowkees/keyball/assets/146462035/98f3c67f-2727-4fdb-be2c-66a45a6376a0)
![20231029_195951](https://github.com/Yowkees/keyball/assets/146462035/164ec0bf-d2ba-4f7e-90c7-89cf48081348)
![20231029_202646](https://github.com/Yowkees/keyball/assets/146462035/2b6fbcb3-5806-4d17-bcb2-fe241b6f1496)
![25](https://github.com/Yowkees/keyball/assets/146462035/2e8e9a6d-0d84-444b-808d-5a32b96512be)

**Comment by Forseti4** [comment]
I have looked at the continuous input state of theZキーの連続入力状態 #331 and tried touching the power section again, but it did not resolve the problem.
![20231029_214412](https://github.com/Yowkees/keyball/assets/146462035/682f01f7-4065-41a8-ad5b-e1c74f031f8e)

The situation with the LED changes each time I input power

![20231029_215336](https://github.com/Yowkees/keyball/assets/146462035/aee518d5-e0f2-4fd4-b5b7-e0d04d31100e)

When I check the input values through a short, they come out as inputs from the left board. I did not solder the left part of the back of the PCB. Also, there are parts that are continuously input when a short occurs in two places

![1qa](https://github.com/Yowkees/keyball/assets/146462035/bcd4fe25-a7a8-453a-b176-44c31043ab37)
![2ws](https://github.com/Yowkees/keyball/assets/146462035/c4c50e9a-b681-470c-92b9-f8d767e3a6b5)

**Comment by Yowkees** [comment]
Hi. Thank you for choosing Keyball61.

You can see circuit data and PCB data here,
https://github.com/Yowkees/keyball/tree/main/keyball61/design_data

Key "DFG" is related to ROW2 pin.
Please check ROW2 is not connected to GND or VCC or other ROW or COL lines.

Why Right side has problem but "DFG" keys are inputed, because "DFG" keys and LEFT judgement key is connected by ROW2.

Anyway, please check ROW2 line.

Thank you.

**Reply by Forseti4** [reply]
I apologize for the novice question. If I suspect a short between ROW2 and VCC, should I check it while applying power? If there is a short with VCC, I'm curious about what measures are needed. An explanation in Japanese is also fine

**Reply by Forseti4** [reply]
If I connect the pins of ROW2 and VCC using a multimeter after applying power, I hear a beep sound. Does this mean that the ROW2 line and VCC line are shorted?

**Reply by Gelotuz** [reply]
I would recommend you to carefully remove the pro micro while it is powered off as it should not be hard because it is in sockets. When pro micro is not connected to pcb and you power it you can see if the same problem occur. If it does you know that problem is in pro micro side. If it doesn't you can narrow it to pcb side. 

While measuring the continuity you can hear sound when there is good connection, also normally in screen you should see value of the ohms. There can be connection in unwanted section even you don't hear sound, you should also check multi-meter monitor if the value is low there might be a problem. Also don't measure continuity while there is power on, continuity measurement acts like jump wire and you might provide power somewhere that may cause damage.

**Reply by Forseti4** [reply]
Thank you for your explanation. I confirmed that there was no input when removed from the PCB. Seeing the same symptoms occur even when I replace it with a pro micro from a board that operates normally, it seems like there is a problem with the PCB. Due to my lack of basic knowledge, I'm not sure what to check and thus, unable to solve the problem

**Reply by Gelotuz** [reply, marked as answer]
You can measure continuity with these points first, Put one probe on the red marked spot and go green marked spots with other probe. All of the points should show infinite value. You can use the not soldered holes for easier probe placement making it easier to keep probe at the place. I marked blue arrow based your japanese post that you had problem also with row 1 for most potential spot for shorted pin.
![probe measurement](https://github.com/Yowkees/keyball/assets/40242562/351246ff-2fe2-4320-a8e1-70cfbc2c077c)

I tried to look possible short spots from gerber files but reading them is too hard for me as the pcb build is so complex.

**Reply by Forseti4** [reply]
Thank you so much for your kind response. During the continuity test, there was no beep, but as you said, when I measured it in ohms, there were parts where the value changed. While removing the pin socket to check, there was some damage. It will take some time to check as I have to order replacement parts. Once I replace the parts and get a result, I will write a reply again. Thank you so much for your help

**Reply by Forseti4** [reply]
Eventually, I was unable to solve the issue and had to recreate it on a new board. The newly made board is working properly. Thank you for thinking through this with me
